stdlib/public/runtime/CMakeLists.txt

```diff
@@ -84,7 +84,7 @@ add_swift_library(swiftRuntime OBJECT_LIBRARY TARGET_LIBRARY
   C_COMPILE_FLAGS ${swift_runtime_library_compile_flags}
   LINK_FLAGS ${swift_runtime_linker_flags}
   INSTALL_IN_COMPONENT never_install
-  TARGET_SDKS ANDROID CYGWIN FREEBSD LINUX)
+  TARGET_SDKS ANDROID CYGWIN FREEBSD LINUX WINDOWS)
```

Comment:
@modocache mulle-objc is unlikely to work with the objc-interop.  There are hardcoded assumptions in the compiler about the ObjC ABI that is being used.  It *MUST* be the non-fragile iOS ABI that is used.